src/store/registration/saga.test.ts

```diff
@@ -508,33 +507,30 @@ describe('authorizeAndCreateWeb3Account', () => {
 describe(createWelcomeConversation, () => {
   function subject(...args: Parameters<typeof expectSaga>) {
     return expectSaga(...args).provide([
-      [matchers.call.fn(getZEROUsersAPI), [{ userId: 'stub-id' }]],
       [matchers.call.fn(createConversation), { id: 'conversation-id' }],
     ]);
   }
   it('creates the welcome conversation between the inviter and new user', async () => {
     const initialState = new StoreBuilder();
+    const inviter = {
+      id: 'inviter-id',
+      userId: 'inviter-id',
+      firstName: 'The inviter',
+      matrixId: '@50c6e12e-1fe2-43f9-8991-ab269696588f:homeserver.domain.com',
+    };
+    initialState.withUsers(inviter);
 
-    await subject(createWelcomeConversation, 'new-user-id', { id: 'inviter-id', matrixId: 'inviter-matrix-id' })
-      .provide([[call(getZEROUsersAPI, ['inviter-matrix-id']), [{ userId: 'inviter-id', firstName: 'The inviter' }]]])
-      .withReducer(rootReducer, initialState.build())
-      .call(createConversation, ['inviter-id'], '', null)
-      .run();
-  });
-
-  it('stores the inviter in state', async () => {
```

Comment:
Removing this test as this is no longer responsible for adding user to Redux. That's handled by getUserByMatrixId and that has it's own tests